client/src/components/Chart.js

```diff
@@ -34,10 +34,11 @@ class CandleStickChartWithCHMousePointer extends React.Component {
 		const xExtents = [start, end];
 
 		return (
+			<div className={'tutorial-step-2-b'}>
```

Comment:
This one too, I don't see a `tutorial-step-2-b` anywhere in the joyride component?

Reply:
yea, thanks for pointing this out. I'll remove it now.